app/scripts/modules/amazon/src/serverGroup/details/AmazonServerGroupActions.tsx

```diff
@@ -220,7 +220,7 @@ export class AmazonServerGroupActions extends React.Component<IAmazonServerGroup
               application={app}
               entityType="serverGroup"
               ownerOptions={entityTagTargets}
-              onUpdate={app.serverGroups.refresh}
+              onUpdate={() => app.serverGroups.refresh()}
```

Comment:
does the `@BindAll` in applicationDataSource obviate the need for this arrow function?

Reply:
nvm, re-read your comment in the PR